tempest/src/test/kotlin/app/cash/tempest/DynamoDbViewTest.kt

```diff
@@ -75,10 +76,10 @@ class DynamoDbViewTest {
     musicTable.albumInfo.save(albumInfo, ifNotExist())
 
     // This fails because the album info already exists.
-    assertThatExceptionOfType(ConditionalCheckFailedException::class.java)
-      .isThrownBy {
-        musicTable.albumInfo.save(albumInfo, ifNotExist())
-      }
+    val exception = assertThrows<AmazonServiceException> {
```

Comment:
Unfortunate consequence of a bug in the DynamoDb Local client where it throws a different exception than what would be thrown in a real-world scenario.